src/sentry/static/sentry/app/views/settings/components/forms/jsonForm.tsx

```diff
@@ -158,16 +206,13 @@ class JsonForm extends React.Component<Props, State> {
       <Box {...otherProps}>
         {typeof forms !== 'undefined' &&
           forms.map((formGroup, i) => (
-            <FormPanel
-              key={i}
-              title={formGroup.title}
-              fields={formGroup.fields}
-              {...formPanelProps}
-            />
+            <React.Fragment key={i}>
+              {this.renderFields({formPanelProps, ...formGroup})}
```

Comment:
Since we're mapping over `forms`, would this make more sense to be called `renderForm`?

Reply:
sure sure